.DEREK.yml

```diff
@@ -9,7 +9,5 @@ features:
   - dco_check
   - comments
   - pr_description_required
-  - release_notes
-  - hacktoberfest
```

Comment:
@maroshii just curiosity, why `hacktoberfest` was in the release branch but not in master? was it deleted in master at some point since we create the branch?

Reply:
Yeah I didn't think it was worth backporting that change. Good thing we hit the conflict so we now understand how it works.
This was the PR: https://github.com/okteto/okteto/pull/2800

Reply:
cool, thanks for the explanation!!